R/step-subset-arrange.R

```diff
@@ -24,8 +24,13 @@ arrange.dtplyr_step <- function(.data, ..., .by_group = FALSE) {
     return(.data)
   }
 
+  no_transmute <- !any(map_lgl(dots, ~ !is_symbol(.x) && !is_call(.x, "-", 1)))
```

Comment:
Maybe use the positive form instead of double negation?

```r
all(map_lgl(dots, ~  is_symbol(.x) || is_call(.x, "-", 1)))
```

Reply:
Ah yep that's better. I'll make the change